drake/systems/plants/test/testIKMoreConstraints.cpp

```diff
@@ -0,0 +1,221 @@
+#include <drake/RigidBodyIK.h>
+#include <drake/RigidBodyManipulator.h>
+#include <drake/RigidBodyConstraint.h>
+
+#include <drake/IKoptions.h>
+#include <iostream>
+#include <cstdlib>
+#include <limits>
+#include <boost/shared_ptr.hpp>
```

Comment:
Boost is not a dependency, hence the failure on windows. shared_ptr is std nowadays, just `#include <memory>`